CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 ## unreleased
 
+### `influx` CLI moved to separate repository
+
+The `influx` CLI has been moved to its [own GitHub repository](https://github.com/influxdata/influx-cli/). Release artifacts
+produced by `influxdb` are impacted as follows:
+
+* Release archives (`.tar.gz` and `.zip`) no longer contain the `influx` binary.
+* The `influxdb2` package (`.deb` and `.rpm`) no longer contains the `influx` binary. Instead, it declares a recommended
+  dependency on the new `influx-cli` package.
+* The `quay.io/influxdb/influxdb` image no longer contains the `influx` binary. Users are recommended to migrate to the
+  `influxdb` image hosted in DockerHub.
+
+With this change, versions of the `influx` CLI and `influxd` server are not guaranteed to exactly match. Please use
```

Comment:
Is it worth mentioning something about compatibility between the CLI versions and the server versions here? Upon reading this, I'd wonder what it might mean to me that the CLI and server versions don't match exactly. The scope of CLI backwards compatibility might not be completely defined yet so it'd make sense to keep it like this as well if that's the case.

Reply:
We haven't formally defined anything yet. My main goal for this section was to prevent people from running `influx version` in bug reports, but I could remove it if you think it's going to cause more confusion than it's worth.

Reply:
Nah that's probably alright in this case.